pkg/jsonSchemaValidator/validator.go

```diff
@@ -20,6 +20,8 @@ func New() *JSONSchemaValidator {
 	return &JSONSchemaValidator{}
 }
 
+var CustomKeyValidationErrorKeyPath = "customKeyValidationErrorKeyPath"
```

Comment:
I see the same peace of code here: https://github.com/datreeio/datree/pull/926/files#r1162390565

I guess one of them is duplicate?